misc/slack-notification/on-start.sh

```diff
@@ -10,3 +10,7 @@ MESSAGE="${ACTION} running \`${ENV0_DEPLOYMENT_TYPE}\` deployment."
 
 __dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 source "${__dir}"/slack.sh "$MESSAGE"
+
+# Making sure we don't have the apply file as we need it to decide if we ran apply or not
+touch apply_ran_successfully
+rm apply_ran_successfully
```

Comment:
```suggestion
rm -f apply_ran_successfully
```

Adding `-f` (force) means no exception when the file doesn't exist, so you don't need the `touch`

Reply:
25be6a83